packages/dhis2/src/Adaptor.js

```diff
@@ -781,6 +907,29 @@ export function dv(dataElement, value) {
   return { dataElement, value };
 }
 
+export function shouldUseNewTracker(resourceType) {
+  return /^(trackedEntityInstances|tracker\/(enrollments|relationships|events|trackedEntities)|enrollments|relationships|events|trackedEntities)$/.test(
+    resourceType
+  );
+}
+
+export function callNewTracker(
+  type = 'update',
+  configuration,
+  options,
```

Comment:
I think `callnewTracker` should set the `importStrategy` option based on the `type`, unless it's already been set.

So if  `type` is `create`, the importStrategy defaults to `CREATE`, etc.

This makes life easier for functions call call into `callNewTracker`, because they don't have to work out the verb AND strategy.